dds/DCPS/ReactorTask.cpp

```diff
@@ -39,23 +38,37 @@ ReactorTask::ReactorTask(bool useAsyncSend)
 }
 
 ReactorTask::~ReactorTask()
+{
+  cleanup();
+}
+
+void
+ReactorTask::cleanup()
```

Comment:
```suggestion
void ReactorTask::cleanup()
```

Rest of the file uses the one line signature style.